tests/core/pyspec/eth2spec/test/capella/block_processing/test_process_withdrawals.py

```diff
@@ -156,8 +165,9 @@ def test_success_max_per_slot(spec, state):
 @with_phases([CAPELLA])
 @spec_state_test
 def test_success_all_fully_withdrawable(spec, state):
+    withdrawal_count = min(len(state.validators), spec.MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP)
```

Comment:
I would frame this as a sanity assert to make it clearer what is being tested

```suggestion
    assert len(state.validators) > spec.MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP
    withdrawal_count = len(state.validators)
    withdrawal_count = min(len(state.validators), spec.MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP)
```

Reply:
we can do this but then this becomes a "mainnet only" test